src/viewdesc.ts

```diff
@@ -971,7 +993,7 @@ class CustomNodeViewDesc extends NodeViewDesc {
   }
 
   destroy() {
-    if (this.spec.destroy) this.spec.destroy()
+    this.spec.destroy?.()
```

Comment:
TypeScript, with our build settings, still produces much worse code for a `?.()` call than this conditional. Could you rever this and also use an `if` in `MarkView.destroy`?

Reply:
Sure. I have reverted in the latest commit.